### ISSUE TITLE
Get update notification

### ISSUE THREAD

**Issue by Cyberavater** [mifi/SimpleInstaBot#107]
As the title says, it'd be nice if I could get a notification when there is a new version available.
I was having a hard time with the bot only to find out there is an update that solves the issue.
Finally thanks for this wonderful tool.

**Comment by mifi** [comment]
was it by any chance this issue? #106
I just fixed that a few hours ago

**Comment by Cyberavater** [comment]
Yes, thank you for fixing it, but It'd be nice if I could get notified about the update within the app itself.
